Cargo.toml

```diff
@@ -2,16 +2,16 @@
 resolver = "2"
 members = [
     "crates/*",
-    "cumulus/client/block-builder",
-    "cumulus/client/cirrus-executor",
-    "cumulus/client/consensus/relay-chain",
-    "cumulus/client/executor-gossip",
-    "cumulus/pallets/*",
-    "cumulus/node",
-    "cumulus/runtime",
-    "cumulus/primitives",
-    "cumulus/test/runtime",
-    "cumulus/test/service",
+    "domains/client/block-builder",
+    "domains/client/cirrus-executor",
+    "domains/client/consensus/relay-chain",
+    "domains/client/executor-gossip",
+    "domains/pallets/*",
+    "domains/node",
+    "domains/runtime",
+    "domains/primitives",
+    "domains/test/runtime",
+    "domains/test/service",
```

Comment:
I feel like wildcards would be helpful here as well, ideally with less nesting (`consensus/relay-chain` -> `consensus-relay-chain` for instance)

Reply:
Make sense, will keep it in mind when doing further cleanup later.